apps/common/views/partials/gatag.html

```diff
@@ -0,0 +1,12 @@
+
+{{#gaTagId}}
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+      ga('create', '{{gaTagId}}', 'auto');
```

Comment:
are we setting a tagId in the locals somewhere? If we need to, please make sure we are.